cron/notification-for-bekrachtigde-mandataris.ts

```diff
@@ -41,6 +41,10 @@ async function handleEffectieveMandatarissen() {
   const mandatarissen =
     await fetchEffectiveMandatarissenWithoutBesluitAndNotification();
 
+  if (mandatarissen.length == 0) {
```

Comment:
why double and not triple? I think it does not mather here as .length will return a number 🤷🏻‍♂️

Reply:
no real reason, just don't really see the added benefit.